rtengine/dcp.cc

```diff
@@ -1178,7 +1178,7 @@ DCPProfile::DCPProfile(const Glib::ustring& filename) :
     // Color Matrix (one is always there)
     if (!md.find(TAG_KEY_COLOR_MATRIX_1)) {
         if (settings->verbose) {
-            std::cerr << "DCP '" << filename.c_str() << "' is missing 'ColorMatrix1'. Skipped." << std::endl;
```

Comment:
@Lawrence37 Wasn't this a fix for something, lately?

Reply:
I had changed printf to cerr in jxl import, but didnt change any instances elsewhere. Changing them in this file now because missing newlines made output unreadable.

Reply:
It's not the `printf()` conversion, that bothers me, but you killed the `.c_str()`...

Reply:
Why do you need `.c_str()` with the stream operators?

Reply:
See 71f452aa1272eec0aa943ae579c6b5417fc204a6.

@Lawrence37 It would be better to add a comment to such odd changes, so they don't creep in again through well-intentioned "fixes".